src/현정빈/Myclass.java

```diff
@@ -0,0 +1,148 @@
+package 현정빈;
+
+import jun.ManagementSystem;
+import jun.Member;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Myclass implements ManagementSystem {
+    HashMap<Integer,Member> map = new HashMap<>(); //컬렉션 중 Map을 선택하여 HashMap 사용
+    // 키는 integer 타입, 값은 Member 타입인 키-값쌍 구조의 map 생성
+
+    /**
+     * 저장소에 멤버를 추가하는 메서드,
+     * 이미 존재하는 id라면 추가에 실패해야 함,
+     * 추가가 성공하면 true를 반환
+     */
+    @Override
+    public boolean addMember(String name, int age, int id) {
+        Member newMember = new Member(name, age, id);
+        if(!map.containsKey(id)) {//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 false라면
+            map.put(id, newMember); //map에 지정된 키(id)와 값(객체 newMember)을 저장
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
+    /**
+     * 저장소에서 멤버를 삭제하는 메서드,
+     * 존재하지 않는 id라면 false를 반환
+     */
+    @Override
+    public boolean removeMember(int id) {
+        if(!map.containsKey(id)){//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 false라면
```

Comment:
```suggestion
        if(!map.containsKey(id)) {//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 false라면
```
if문의 조건 로직 이후에 오는 중괄호는 공백을 통해 구분해주는 것이 관례에요
관례라 함은 전세계 개발자들이 99%가 이런 식으로 컨벤션을 맞춰주고 있다고 생각하면 편해요
이런 관례들에 시간이 지나면서 익숙해지실거에요, 앞으로 스터디하면서 익혀봐요!